hyperSpec/vignettes/fileio.Rmd

```diff
@@ -1259,8 +1259,9 @@ to stop with an error, but it is polite to issue an additional warning.
 writeLines(readLines("read.txt.PerkinElmer.R"))
 ```
 
+imports the spectra.
 Note how the labels are set.
-The label with the special name `.wavelength`{.r} corresponds to the wavelength axis, all data columns should have a label with the same name.
+The label with the unique name `.wavelength`{.r} corresponds to the wavelength axis, all data columns should have a label with the same name.
```

Comment:
Word "special" (or its synonym) is more appropriate here as `.wavelength` is for `@wavelength` slot of `hyperSpec` object. Maybe "certain" is more appropriate here. Word "unique" does not reflect the essence of the meaning.